From a4d3fa64e7ec479a1311300ab9c1b435e4fd64b3 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@nic.cz>
Date: Wed, 21 Nov 2018 15:25:57 +0100
Subject: [PATCH 11/12] python3: Drop pytest from requirements

We don't plan to run them anyway, so let's keep this dependency out.

Signed-off-by: Michal Hrusecky <michal.hrusecky@nic.cz>
---
 lang/python/python3-package.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/lang/python/python3-package.mk b/lang/python/python3-package.mk
index 7f47370..0986537 100644
--- a/lang/python/python3-package.mk
+++ b/lang/python/python3-package.mk
@@ -141,8 +141,12 @@ endef
 
 Py3Build/Compile=$(Py3Build/Compile/Default)
 
+define python3_drop_pytest_requirements
+	sed -i '/^pytest/ d' $(PKG_BUILD_DIR)/requirements.txt || true
+endef
 
 Hooks/Compile/Post += python3_fix_interpreter_paths
+Hooks/Compile/Pre += python3_drop_pytest_requirements
 
 ifeq ($(BUILD_VARIANT),python3)
 define Build/Compile
-- 
2.19.1

